addons/l10n_es_edi_tbai/models/account_move.py

```diff
@@ -188,6 +188,15 @@ def _l10n_es_tbai_lock_move(self):
     # WEB SERVICE CALLS
     # -------------------------------------------------------------------------
 
+    def l10n_es_tbai_resend_bill(self):
+        self.ensure_one()
+        self.l10n_es_tbai_post_document_id = False
+        error = self.with_context(batuz_correction=True)._l10n_es_tbai_post()
+        if error:
```

Comment:
```suggestion
        if error := self.with_context(batuz_correction=True)._l10n_es_tbai_post():
```
Good case for walrus operator.

Reply:
I will adapt, but I am not sure if it is the most natural use of Walrus operator.